src/api/DesktopAgent.ts

```diff
@@ -375,6 +376,27 @@ export interface DesktopAgent {
    */
   addContextListener(contextType: ContextType | null, handler: ContextHandler): Promise<Listener>;
 
+  /** 
+   * Register a handler for events from the Desktop Agent. Whenever the handler function 
+   * is called it will be passed an event object with details related to the event.
+   * 
+   * ````js
+   * // any event type
+   * const listener = await fdc3.addEventListener(null, event => { ... });
+   * 
+   * // listener for a specific event type that logs its details
+   * const userChannelChangedListener = await fdc3.addEventListener(FDC3EventType.USER_CHANNEL_CHANGED, event => { 
+   * console.log(`Received event ${event.type}\n\tDetails: ${event.details}`);
+   * //do something else with the event
+   * });
+   * ````
```

Comment:
```suggestion
   * ```js
   * // any event type
   * const listener = await fdc3.addEventListener(null, event => { ... });
   * 
   * // listener for a specific event type that logs its details
   * const userChannelChangedListener = await fdc3.addEventListener(FDC3EventType.USER_CHANNEL_CHANGED, event => { 
   * console.log(`Received event ${event.type}\n\tDetails: ${event.details}`);
   * //do something else with the event
   * });
   * ```
```